ethcore/src/evm/interpreter/mod.rs

```diff
@@ -292,14 +292,21 @@ impl<Cost: CostType> Interpreter<Cost> {
 					}
 				};
 			},
-			instructions::CALL | instructions::CALLCODE | instructions::DELEGATECALL => {
+			instructions::CALL | instructions::CALLCODE | instructions::DELEGATECALL | instructions::STATICCALL => {
```

Comment:
imo, it actually should be the same as `DELEGATECALL`, since we are not creating new accounts, nor transfering value